matching/pom.xml

```diff
@@ -31,6 +31,14 @@
     </repository>
   </repositories>
 
+  <properties>
+    <scala.majorVersion>2.12</scala.majorVersion>
+    <scala.minorVersion>18</scala.minorVersion>
+    <scala.version>${scala.majorVersion}.${scala.minorVersion}</scala.version>
+    <spotless.version>2.41.1</spotless.version>
+    <scalafmt.version>3.7.17</scalafmt.version>
+  </properties>
+
```

Comment:
As far as I understand, all the properties are automatically inherited from the parent POM, so we don't need to redeclare them here.